PayloadPkg/FirmwareUpdate/GetCapsuleImage.c

```diff
@@ -427,41 +428,48 @@ GetCapsuleImage (
     DEBUG((DEBUG_ERROR, " CapsuleInfo not found \n"));
     return EFI_NOT_FOUND;
   }
+  DevInstanace_configured = CapsuleInfo->DevInstance;
+  //DevInstance is enumerated upto configured DevInstance in "CfgData_CapsuleInformation.yaml"
+  for (CapsuleInfo->DevInstance = 0; CapsuleInfo->DevInstance <= DevInstanace_configured; CapsuleInfo->DevInstance ++)
+  {
+    Status = InitBootDevice(CapsuleInfo);
+    if (EFI_ERROR(Status)) {
+      return Status;
```

Comment:
And if we supports multiple device, should try next one if there is error?

Reply:
No, It will increment only the instance of configured device and not multiple device

Reply:
For example, if there are 2 NVMe instances, it means the system could have 2 NVMe devices installed. When it tried the first instance/device failed, it should try another instance/device, right?

Reply:
should we close this PR if it is not required?